.github/workflows/continuous-integration.yml

```
@@ -38,10 +37,8 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - uses: actions-rs/toolchain@v1
-        # No wheels exist for orjson on Python 3.10 on MacOS or Windows, and the Linux
-        #  wheels are sometimes not uploaded until after the package is initially
-        # published. This sets up the Rust nightly toolchain so we can build the orjson
-        # wheel.
+        # Wheels for orjson on Python 3.10 are still spotty on MacOS or Windows. This sets up the Rust nightly
```

```suggestion
        # Wheels for orjson on Python 3.10 are still spotty on MacOS or Windows. This sets up the Rust
```

Turns out we're using the [stable toolchain](https://github.com/stac-utils/pystac/pull/656/files#diff-c471496bcb3ed47397b73d2e3e3aa41a8a679c86accaed9e05676fd9d5987434R44).

---

Good catch. As of [v0.3.2] `orjson` compiles on the `stable` toolchain for Rust >= 1.54.0, but I forgot to update that comment.